collectd_plugin/collectd_plugin.py

```diff
@@ -397,6 +400,9 @@ def read(data):
     correctness.
     """
 
+    if not data.get("monitor_endpoint"):
+         return
```

Comment:
it looks like we always configure `data['metrics_queue']` so this is probably what we want